templates/api/knowledge-panels/health/nutriscore/nutriscore.tt.json

```diff
@@ -24,7 +25,26 @@
                 }
             },
             [% END %]
-        [% END %]            
+        [% END %]
+        // Display actions if we are missing data to compute the Nutri-Score
+        [% IF panel.nutriscore_unknown_reason %]
+        {
+            "element_type": "action",
+            "action_element": {
+                "html": `[% lang("actions_to_compute_nutriscore") %]`,
+                "actions": [
+                    [% IF panel.nutriscore_unknown_reason == "missing_category" %]
+                        "add_categories"
+                    [% ELSIF panel.nutriscore_unknown_reason == "missing_nutrition_data" %]
+                        "edit_nutrition"
```

Comment:
Is it intentional to have "edit_nutrition" instead of "add_nutrition_facts" ?

Reply:
It's not intentional, at first I thought it would be useful to distinguish the two, then changed my mind. I'll fix it.